packages/compass-e2e-tests/helpers/commands/do-connect.js

```diff
@@ -1,21 +1,23 @@
 const Selectors = require('../selectors');
 
-module.exports = function (app) {
+module.exports = function (compass) {
   return async function doConnect(timeout, expectSuccess = true) {
-    const { client } = app;
-    await client.clickVisible(Selectors.ConnectButton);
-    if (expectSuccess) {
+    const { browser } = compass;
+    await browser.clickVisible(Selectors.ConnectButton);
+    let selector;
+    if (expectSuccess === 'either') {
```

Comment:
If something goes from true/false to true/false/doesn't matter, what's the best value/name to use?

I introduced this because I tend to run mongodb on localhost for testing and the SRV test connects to that same port so in some environments it would connect successfully and in others it wouldn't.. but it doesn't matter because the test only cares about how it resolved.

Interestingly the same happened in CI on windows before I ultimately disabled the test on windows 🤔

Reply:
Do we have a case where we explicitly expect connection to fail? If we don't, maybe it's worth making a `false` case just a catch all that waits for a connection or a failure state, similar to `Promise.allSettled` it can return an error in the case of a failure (if we even care about the error message)

If you're feeling like keeping it as it is right now, maybe changing this to something like `connectionStatus: 'success' | 'error' | 'either'` would be a bit more explicit about the intention?

Reply:
Going with connectionStatus. Don't think we currently have one that expects error but we should probably have a test for that. Also not sure what's going to happen to the test that's currently skipped on windows when we fix it.